google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -31,6 +31,7 @@
 
 from google.cloud.aiplatform.compat.types import (
     pipeline_job as gca_pipeline_job,
+    pipeline_job_v1beta1 as gca_pipeline_job_v1beta1,
```

Comment:
Let's wait until https://github.com/googleapis/python-aiplatform/pull/1221 is merged so you can change this to use `v1`.

Reply:
Done.